src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -10,8 +10,10 @@
 /**
  * Keep reads with mapping qualities within a specified range.
  *
- * Note: this filter does not handle specially the unavailable mapping quality ({@link org.broadinstitute.hellbender.utils.QualityUtils#MAPPING_QUALITY_UNAVAILABLE}).
- * Use {@link org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.MappingQualityAvailableReadFilter} to explicitly filter out reads with unavailable quality.
+ * <p>Note: this filter is not designed to handle the unavailable mapping quality (255).
+ * Use MappingQualityAvailableReadFilter to explicitly filter out reads with unavailable quality.
+ * @see org.broadinstitute.hellbender.utils.QualityUtils#MAPPING_QUALITY_UNAVAILABLE
+ * @see org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.MappingQualityAvailableReadFilter
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
```

Comment:
Oh I naively thought those would be converted to links to the relevant html page. If it's strictly javadocs then we don't want that. What I'd like to be able to do is write anywhere in a description "see [this thing](link_to_html_of_thing) for details", preferably without having to figure out the url myself. Does that make sense?